pagina5.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pagina 5</title>
+    <link rel="stylesheet" href="style2.css">
+    <style>
+        .second-paragraph{
+            font-weight: bold;
+        }
+    </style>
+</head>
+<body>
+    <div>
+        <div>
+            <img src="computerlogo2.jpg" title="Computer" class="img2">        
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec imperdiet lorem sit amet molestie rutrum. Vestibulum facilisis velit tempor libero luctus consequat. Ut vel elit sed nulla pulvinar pellentesque et at lorem. Quisque dignissim pharetra dolor quis aliquam. Nulla bibendum venenatis leo, ut ornare risus dignissim nec. Praesent vestibulum mauris cursus suscipit dictum. Sed ut lorem et est vehicula condimentum quis vel ex. Duis fermentum finibus erat, vitae mattis libero.
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec imperdiet lorem sit amet molestie rutrum. Vestibulum facilisis velit tempor libero luctus consequat. Ut vel elit sed nulla pulvinar pellentesque et at lorem. Quisque dignissim pharetra dolor quis aliquam. Nulla bibendum venenatis leo, ut ornare risus dignissim nec. Praesent vestibulum mauris cursus suscipit dictum. Sed ut lorem et est vehicula condimentum quis vel ex. Duis fermentum finibus erat, vitae mattis libero.
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec imperdiet lorem sit amet molestie rutrum. Vestibulum facilisis velit tempor libero luctus consequat. Ut vel elit sed nulla pulvinar pellentesque et at lorem. Quisque dignissim pharetra dolor quis aliquam. Nulla bibendum venenatis leo, ut ornare risus dignissim nec. Praesent vestibulum mauris cursus suscipit dictum. Sed ut lorem et est vehicula condimentum quis vel ex. Duis fermentum finibus erat, vitae mattis libero.
+        </div>
+        <div class="second-paragraph">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec imperdiet lorem sit amet molestie rutrum. Vestibulum facilisis velit tempor libero luctus consequat. Ut vel elit sed nulla pulvinar pellentesque et at lorem. Quisque dignissim pharetra dolor quis aliquam. Nulla bibendum venenatis leo, ut ornare risus dignissim nec. Praesent vestibulum mauris cursus suscipit dictum. Sed ut lorem et est vehicula condimentum quis vel ex. Duis fermentum finibus erat, vitae mattis libero.
+        </div>
```

Comment:
This paragraph is not linked to others and should be placed under the image. (Don't change html structure, only css)